### PR TITLE
Fix for an issue where plugins that transform API response

### PR DESCRIPTION
**Pull request by mdobson**
Fix for part of #17

### CODE REVIEW

**Comment by mdobson** [comment]
Added some commentary. Linking to outside documentation in this case is overkill.

File: lib/plugins-middleware.js [hunk 1]
```diff
@@ -527,7 +527,7 @@ function _executePluginHandler(plugin, type, options) {
           }
         };
         plugin['on' + handler](options.sourceRequest, options.sourceResponse,
-          plugin['on' + handler].length === 3 ? fx : pluginOptions, //if 3 args must be the older style
+          plugin['on' + handler].length === 3 ? fx : pluginOptions.data, //if 3 args must be the older style
```
Comment (F1ERRO) [inline lib/plugins-middleware.js]: this parameter can just be `data` and the `pluginOptions` object can be deleted

Comment (F1ERRO) [inline lib/plugins-middleware.js]: this parameter can just be `data` and the `pluginOptions` object can be deleted

**Comment by mdobson** [comment]
updated

**Pull request merged**